multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/query/providers/SqlOperationLogQueryProvider.java

```diff
@@ -23,6 +27,30 @@ public class SqlOperationLogQueryProvider {
     private static final String SELECT_LOGS_BY_SPACE_ID_OPERATION_ID_AND_OPERATION_LOG_NAME = "SELECT ID, OPERATION_LOG, OPERATION_LOG_NAME FROM process_log WHERE SPACE=? AND OPERATION_ID=? AND OPERATION_LOG_NAME=? ORDER BY MODIFIED ASC";
     private static final String SELECT_LOGS_BY_SPACE_ID_AND_NAME = "SELECT DISTINCT ID, OPERATION_LOG, OPERATION_LOG_NAME, MODIFIED FROM process_log WHERE SPACE=? AND OPERATION_ID=? ORDER BY MODIFIED ASC";
 
+    public static void saveLogInDatabase(OperationLogEntry operationLogEntry, DataSource dataSource) {
+        try (PreparedStatement statement = dataSource.getConnection()
+                                                     .prepareStatement(INSERT_FILE_ATTRIBUTES_AND_CONTENT)) {
+
+            statement.setString(1, operationLogEntry.getId());
+            statement.setString(2, operationLogEntry.getSpace());
+
+            if (operationLogEntry.getNamespace() == null) {
+                statement.setNull(3, Types.NULL);
+            } else {
+                statement.setString(3, operationLogEntry.getNamespace());
+            }
+
+            statement.setTimestamp(4, Timestamp.valueOf(operationLogEntry.getModified()));
+            statement.setString(5, operationLogEntry.getOperationId());
+            statement.setString(6, operationLogEntry.getOperationLog());
+            statement.setString(7, operationLogEntry.getOperationLogName());
+
+            statement.executeQuery();
+        } catch (SQLException e) {
+            throw new OperationLogStorageException(Messages.FAILED_TO_SAVE_OPERATION_LOG_IN_DATABASE, e);
+        }
+    }
+
     public SqlQuery<List<OperationLogEntry>> getListFilesQueryBySpaceAndOperationId(String space, String operationId) {
```

Comment:
either make all methods static or use the fact that they aren't static and this class would have an instance